app/views/lit/localizations/_form.html.erb

```diff
@@ -19,5 +19,32 @@
 
   <button class="btn btn-primary btn-sm" type="submit"><%= I18n.t('lit.common.update', default: "Update") %></button>
   <button class="btn btn-default btn-sm cancel" type="reset"><%= I18n.t('lit.common.cancel', default: 'Cancel') %></button>
+
+  <% if Lit::Cloud.provider %>
```

Comment:
I think that `Lit::Cloud` name is slightly misleading here, how about `Lit::CloudTranslations`?